app/Http/Controllers/Admin/AdminController.php

```diff
@@ -15,21 +39,143 @@ class AdminController extends Controller
      */
     public function index()
     {
-        return view('admin.index');
+        $roleAdmin = \App\User::ADMIN;
+        return view('admin.user.dashboard', ['roleAdmin' => $roleAdmin]);
     }
 
+    /**
+     * Log account admin out of the application.
+     *
+     * @return void
+     */
     public function userList()
     {
-        return view('admin.user.list');
+        $users = Auth::user();
+        $roleAdmin = \App\User::ADMIN;
```

Comment:
the variable $roleAdmin repeat alot , please put is class variable

Reply:
Fixed, I have added the function roleAdmin for this case